lib/ts/recipe/thirdparty/components/themes/styles.css

```diff
@@ -9,6 +9,10 @@
     padding-bottom: 9px;
 }
 
+[data-supertokens~="providerContainer"] button {
+    cursor: pointer;
```

Comment:
can you confirm that if the user provides their own custom component (with a custom provider, or with the google provider), the pointer style is still applied?

Reply:
it will be applied if user uses button element

Reply:
The user is not supposed to use the button element in their custom component cause the click handler is applied by us. Please make sure you understand how things work